### ISSUE TITLE
 StayKit

### ISSUE THREAD

**Issue by GrayProphet** [0xthirteen/StayKit#2]
Hi, in the Non-Elevated: Scheduled Task and Elevated: Scheduled Task methods, the link goes to the same function. Is that what you intended?

**Comment by 0xthirteen** [comment]
yeah, it probably should haven't been separated by elevated and non-elevated but gone into misc because the only real difference is the runas user that is passed in 
https://github.com/0xthirteen/StayKit/blob/master/StayKit.cna#L694
https://github.com/0xthirteen/SharpStay/blob/master/SharpStay/SharpStay/Program.cs#L1264
